website/src/css/quotecard.module.css

```diff
@@ -0,0 +1,53 @@
+.quotecard {
+  background: #fff;
+  color: #222222;
+  padding: 20px;
+  padding-left: 50px;
+  box-sizing: border-box;
+  box-shadow: 12px 12px 2px 1px var(--ifm-color-primary-lightest);
```

Comment:
```suggestion
  box-shadow: 5px 5px 15px 1px var(--ifm-color-primary-lightest);
```
J'aime bien l'idée des quotes dans le coin droit :smiley: , l'ombre n'est peut-être pas assez dégradée, je l'ai modifiée un peu pour voir :

![newsletter-comments](https://user-images.githubusercontent.com/38593370/116272323-1af7cf80-a781-11eb-8da1-710937e390dd.png)
![newsletter-comments-dark](https://user-images.githubusercontent.com/38593370/116272327-1b906600-a781-11eb-8007-571c23782c24.png)